lib/model/store.dart

```diff
@@ -353,6 +359,22 @@ class PerAccountStore extends ChangeNotifier with StreamStore {
     super.dispose();
   }
 
+  void getMessageUsers(Message message) {
+    if (message is StreamMessage) {
+      Set<int> userIds = _streamUserIds[message.streamId] ?? {};
+      if (message.senderId == selfUserId) return;
+      userIds = {message.senderId}..addAll(userIds);
```

Comment:
This copies the Set. Better to mutate it instead — that has cost O(1), rather than O(N) in the size of the set.